--- D:\GAMEZ\Minecraft\_Z@RYA Vanilla 1.18.2\ПРОДАКШН\Z@RYA (Vanilla 1.18.2)\.minecraft\config\fancymenu\config.txt Thu Jun 30 22:12:11 2022
+++ D:\GAMEZ\Minecraft\_Z@RYA Vanilla 1.18.2\ПРОДАКШН\Z@RYA (Vanilla 1.18.2)\.minecraft\config\fancymenu\config.txt Thu Jun 30 22:23:45 2022
@@ -16,7 +16,7 @@
 ##[customization]
 
 B:advancedmode = 'false';
-B:showcustomizationbuttons = 'true';
+B:showcustomizationbuttons = 'false';
 
 
 ##[loading]
